src/gui/widgets/TempoSyncKnob.cpp

```diff
@@ -291,7 +291,12 @@ void TempoSyncKnob::showCustom()
 	if( m_custom == NULL )
 	{
 		m_custom = new MeterDialog( gui->mainWindow()->workspace() );
-		gui->mainWindow()->addWindowedWidget( m_custom );
+		QMdiSubWindow * m_subWindow;
```

Comment:
You should remove the "m_" prefix since this is not a member variable.

Reply:
Also, you can consolidate the declaration and the assignment as well.